src/containers/MyNdla/Folders/FolderActions.tsx

```diff
@@ -43,14 +48,20 @@ const FolderActions = ({ selectedFolder, setFocusId, folders, inToolbar = false,
   const { deleteFolder } = useDeleteFolderMutation();
   const { addFolder } = useAddFolderMutation();
   const { updateFolderStatus } = useUpdateFolderStatusMutation();
+  const { unFavoriteSharedFolder } = useUnFavoriteSharedFolder();
 
   const { user, examLock } = useContext(AuthContext);
 
   const shareRef = useRef<HTMLButtonElement | null>(null);
   const unShareRef = useRef<HTMLButtonElement | null>(null);
+  const unLinkRef = useRef<HTMLButtonElement | null>(null);
 
   const isFolderShared = selectedFolder?.status !== "private";
 
+  const refreshPage = () => {
+    window.location.reload();
+  };
```

Comment:
Virker veldig hardt å laste sida på nytt berre for å slette ei mappe. Oppdatering av cache burde gjøre samme nytten.